org-formation/050-costs/categories.yaml

```diff
@@ -6,4 +6,114 @@ Resources:
     Properties:
       Name: 'Program Code'
       RuleVersion: 'CostCategoryExpression.v1'
-      Rules: !ReadFile program-code-rules.json
+      Rules:
+        Fn::Transform:
+          - Name: SageCostRuleGenerator
+        # Account names can be found at https://d-906769aa66.awsapps.com/start#/
+        RegularValues:
```

Comment:
I assume AWS CFN will trigger `SageCostRuleGenerator` macro when changes are made to the below configs?  I would definitely test to make sure that happens.

Reply:
I'm not sure I understand the question, but CFN will run the macro every time it processes the template whether or not it has changed, though https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/template-macros.html doesn't really specify when that processing happens